packages/block-editor/src/components/block-settings/container.native.js

```diff
@@ -41,6 +44,13 @@ function BottomSheetSettings( {
 				>
 					<InspectorControls.Slot />
 				</BottomSheet.NavigationScreen>
+
+				<BottomSheet.NavigationScreen
+					name={ InspectorControlsChild.screenName }
+				>
```

Comment:
This would be the one area I see conflicting with the use cases I have worked on thus far. The focal point picker requires a full-height bottom sheet, much like [the link picker](https://github.com/WordPress/gutenberg/blob/44573cef95cd4e3f08816a3ba5383302543d5b25/packages/block-editor/src/components/block-settings/container.native.js#L59-L67) (which also leverage `isScrollable`).

I wonder if there is anyway for us to allow navigating to a child sheet for both auto-height and full-height? Alternatively, would it be appropriate to set up two, separate `InspectorControlsChild`s for each use case with different names (e.g. `'InspectorControlsChild'`, `'InspectorControlsChildFullHeight'`)?

That said, it may be premature to attempt to tackle both the `fullScreen` and `isScrollable` configurations. Maybe those use cases are different enough that they should not use `InspectorControlsChild`. We could start with this abstraction as-is and explore from there?

Reply:
I managed to add support for fullScreen to the new `InspectorControlsSubSheet` component props to @mkevins for suggesting it. 

I think we should be able to add support for `isScrollable` but this PR is getting pretty big as it is. So I might try to do this in a new PR in which we could try to see if the linkedControl could be done this way. 

We could do it by adding `shouldEnableBottomSheetScroll( true )` in the `InspectorControlsSubSheet` component.